sacred/utils.py

```diff
@@ -89,10 +89,15 @@ class TimeoutInterrupt(SacredInterrupt):
 
 
 class SacredError(Exception):
-    def __init__(self, message, print_traceback=True, filter_traceback=True,
-                 print_usage=False):
+    def __init__(self, message, print_traceback=True,
+                 filter_traceback='default', print_usage=False):
         super(SacredError, self).__init__(message)
         self.print_traceback = print_traceback
+        if filter_traceback not in ['always', 'default', 'never']:
+            raise ValueError(
+                'filter_traceback must be one of \'always\', '
```

Comment:
I would replace the outer quotation marks with `"` to avoid escaping inside the string.

Reply:
I usually prefer to use the `'` strings, or at least use the same type of quotation marks for the whole code. Since the sacred code seems to randomly mix `'` and `"`, it does not make a difference and I could change it. I could as well replace the single quotation marks `\'` with double ones `"`, or completely remove them. In the rest of the code of sacred I can find all versions ([single quotes in double quotes](https://github.com/IDSIA/sacred/blob/6fb05b1ee1b7706b44c3ebd852e1e234841ea2ce/sacred/commandline_options.py#L279), [double quotes in single quotes](https://github.com/IDSIA/sacred/blob/6fb05b1ee1b7706b44c3ebd852e1e234841ea2ce/sacred/commandline_options.py#L299) and [no quotation marks](https://github.com/IDSIA/sacred/blob/6fb05b1ee1b7706b44c3ebd852e1e234841ea2ce/sacred/utils.py#L655)). Which one do you prefer?

Reply:
Yeah quotations are a mess. I just meant that if you want to have one type of quotation marks inside a string I always prefer to use the other type as outer quotation marks to avoid escaping, which in my view makes it more readable.